flink-table/flink-table-api-scala-bridge/pom.xml

```diff
@@ -122,6 +122,42 @@ under the License.
 					</execution>
 				</executions>
 			</plugin>
+
+			<!-- Adding scala source directories to build path
+			 	 This is required for the source jar -->
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>build-helper-maven-plugin</artifactId>
+				<executions>
+					<!-- Add src/main/scala to build path -->
+					<execution>
+						<id>add-source</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>add-source</goal>
+						</goals>
+						<configuration>
+							<sources>
+								<source>src/main/scala</source>
+							</sources>
+						</configuration>
+					</execution>
+					<!-- Add src/test/scala to build path -->
+					<execution>
+						<id>add-test-source</id>
+						<phase>generate-test-sources</phase>
+						<goals>
+							<goal>add-test-source</goal>
+						</goals>
+						<configuration>
+							<sources>
+								<source>src/test/java</source>
```

Comment:
Do we need this `src/test/java` ? I though we've added them by `maven-jar-plugin:test-jar`.

Reply:
In this case no. Probably only required if the test sources include both Scala _and_ Java code.